src/main/resources/static/css/main.css

```diff
@@ -1,59 +1,58 @@
+/* Header */
 h1 {
     color: White;
     padding-bottom: 20px;
-
 }
-#zipsearch{
+#zipSearch{
     padding-left: 20px;
     align-content: center;
 }
 
+#searchButton {
+    color: lightskyblue;
+    margin-left: 10px;
+    border-color: lightskyblue;:
+}
+
+#searchButton:hover{
+    background-color:ghostwhite;
+}
+
+.btn-primary {
+
+    background-color: #007bff;
+    border-color: ghostwhite;
+}
+
+/* Body */
+#body{
+    height: 100%;
+}
```

Comment:
What is the technical debt reduction of this change?  Indicate that here, and cite best practices.